ignite/pkg/events/events_test.go

```diff
@@ -15,23 +15,31 @@ func TestBusSend(t *testing.T) {
 	}{
 		{
 			name: "send status ongoing event",
-			bus:  make(Bus),
+			bus:  Bus{evchan: make(chan Event)},
```

Comment:
I think we should always use New for it, setting private fields prevents us making black box unit testing.